src/services/data/DataService.tsx

```diff
@@ -7,11 +8,16 @@ export class DataService {
   static postMultipartRequest(data: any, fullURI?: string): Promise<any> {
     const formData = new FormData();
     if (fullURI === undefined) {
-      fullURI = process.env.REACT_APP_MCP_DATA_SOURCE + "files";
+      fullURI =
+        process.env.REACT_APP_MCP_DATA_MAPPING_UTIL + "api/excel/jsontoexcel";
     }
-    Object.keys(data).forEach((name: string) =>
-      formData.append(name, data[name])
-    );
-    return fetch(fullURI, Call.postCall(formData));
+    formData.append("files", data.file);
+    return fetch(fullURI, Call.postForm(formData)).then(response => {
+      return response.json().then((data: any) => ({
+        status: response.status,
+        message: data.message,
+        fileReport: data.files
+      }));
+    });
```

Comment:
A flatter (like less nesting) way of doing this could look like this:

```js
return fetch(fullURI, Call.postForm(formData)).then(response => {
  return response.json();
}).then(data => ({
  status: response.status,
  message: data.message,
  fileReport: data.files
}));
```

Reply:
Won't have access to `response.status` with this approach.